lib/api/test/unit/units/validateHeaders.test.js

```diff
@@ -0,0 +1,139 @@
+const { assert } = require('chai');
+const { validateHeaders } = require('../../../units/validateHeaders');
+
+describe('validateHeaders', () => {
+  describe('given matching headers', () => {
+    const res = validateHeaders(
+      {
+        headers: {
+          'content-type': 'application/json',
+          connection: 'keep-alive'
+        }
+      },
+      {
+        headers: {
+          'content-type': 'application/json',
+          connection: 'keep-alive'
+        }
+      }
+    );
+
+    it('has proper validator', () => {
```

Comment:
I'd be specific: "has HeadersJsonExample validator" or "has application/vnd.apiary.http-headers+json as the real type"

Code sometimes changes and it's not clear what was the original intention with ambiguous words like "has proper this" or "has expected that" or "has correct this". Also, when running the tests, such description are not very informative in the terminal output.

Reply:
Thanks for pointing out, some of the initial tests were left behind. I've updated all test suits to contain explicit intent.